src/test/java/gov/healthit/chpl/aqa/pageObjects/ChplAPIPage.java

```diff
@@ -0,0 +1,46 @@
+package gov.healthit.chpl.aqa.pageObjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Class ChplAPIPage definition.
+ */
+public final class ChplAPIPage {
+    private ChplAPIPage() {}
+    private static WebElement element = null;
+
+     /**
+     * Controller link.
+     * @param driver WebDriver
+     * @return controller link element
+     * @param controller desired controller to open
+     */
+    public static WebElement controllerLink(final WebDriver driver, final String controller) {
+        element = driver.findElement(By.xpath("//*[@id=\"" + controller + "*\"]/li/h4/a"));
```

Comment:
```suggestion
        return driver.findElement(By.xpath("//*[@id=\"" + controller + "*\"]/li/h4/a"));
```
Can delete the next line, and do the same changes for the other PageObject elements